collect_app/src/main/res/layout/show_qrcode_fragment.xml

```diff
@@ -81,6 +63,10 @@
             android:layout_weight="1"
             android:padding="5dp"
             android:textSize="15sp"
-            android:textStyle="bold" />
+            android:gravity="center_vertical"
+            android:text="@string/qrcode_with_both_passwords"
+            android:textColor="#000000"
```

Comment:
with the dark color scheme, white text over a yellowish background: 
https://github.com/hypercubestart/collect/blob/51fbeda54ab3c3510e135114a660c2fe0fde2be8/collect_app/src/main/res/layout/show_qrcode_fragment.xml#L43
did not look well. So, I planned to just keep the color scheme consistent. Is this okay? As a sidenote, should I pull out the color hex values into `colors.xml`

Reply:
Ah right I see. You're right that is not going to look good on the yellow! We really want our colors to as much as possible come from our theme attributes. This makes them easier to change but also just limits the amount of colors floating around in the app. I think for the background color here we should use `?colorError` (a red warning/error color) instead of the yellow and for the text we should use `?colorOnError`.

Reply:
i can roll with this, the error icon will also use the `?colorOnError` to match the color of the text